ext/opentelemetry-ext-requests/src/opentelemetry/ext/requests/__init__.py

```diff
@@ -50,9 +50,15 @@
 from opentelemetry import context, propagators, trace
 from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
 from opentelemetry.ext.requests.version import __version__
-from opentelemetry.trace import SpanKind, get_tracer
+from opentelemetry.trace import SpanKind
 from opentelemetry.trace.status import Status, StatusCanonicalCode
 
+# StackDriver exporter spins up a new thread (that doesn't inherit the
+# "suppress_instrumentation" context) that makes a request call. We need to
+# manually blacklist the url to avoid falling into an infinite loop.
+# https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/issues/3
+SUPPRESSION_BLACKLIST = ["https://oauth2.googleapis.com/token"]
```

Comment:
this feels like the wrong approach to this problem for me. This is explicitly adding vendor-specific blacklists into an instrumentation. 

In the future these instrumentations may not live in the same repo, or be owned by the same groups. Maybe we can talk more in the linked issue, but I don't think this is an acceptable workaround for the project.

Reply:
Removing this